<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLSLiveInfoChzzk</class>
 <widget class="QWidget" name="PLSLiveInfoChzzk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LiveInfo.liveinformation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="PLSLiveInfoDualWidget" name="dualWidget" native="true"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>350</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QWidget" name="thumWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="PLSSelectImageButton" name="thumbnailButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignTop">
           <widget class="QLabel" name="thumDetailLabel">
            <property name="text">
             <string>chzzk.thum.tip</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="midwidget" native="true">
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="rowWrapPolicy">
           <enum>QFormLayout::DontWrapRows</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>27</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>14</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="liveTitleLabel">
            <property name="text">
             <string>LiveInfo.base.Title</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="PLSLineEdit" name="lineEditTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="placeholderText">
             <string>chzzk.title.placeholder</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="leftTitle_category">
            <property name="text">
             <string>LiveInfo.Base.Category</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="PLSSearchCombobox" name="lineEditCategory">
            <property name="placeholderText">
             <string>LiveInfo.Twitch.Category.Placeholder</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="leftTitle_age">
            <property name="text">
             <string>chzzk.age.limit</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="PLSCheckBox" name="age_checkbox" native="true">
               <property name="text" stdset="0">
                <string>chzzk.age.limit.content</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignLeft">
              <widget class="QPushButton" name="pushButton_age">
               <property name="text">
                <string>Notice.Bottom.Learn.Button.Text</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="leftTitle_money">
            <property name="text">
             <string>chzzk.need.money</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="PLSCheckBox" name="money_checkbox" native="true">
            <property name="text" stdset="0">
             <string>chzzk.need.money.content</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="leftTitle_chat">
            <property name="text">
             <string>chzzk.chat.limit</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QWidget" name="ageRadioWidget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="PLSRadioButton" name="radioButton_chat_all" native="true">
               <property name="text" stdset="0">
                <string>chzzk.chat.limit.all</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>18</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="PLSRadioButton" name="radioButton_chat_follow" native="true">
               <property name="text" stdset="0">
                <string>chzzk.chat.limit.followers</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="PLSHelpIcon" name="label">
               <property name="toolTip">
                <string>chzzk.chat.limit.followers.tooltip</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>18</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="PLSRadioButton" name="radioButton_chat_manager" native="true">
               <property name="toolTip">
                <string>chzzk.chat.limit.admin.tooltip</string>
               </property>
               <property name="text" stdset="0">
                <string>chzzk.chat.limit.admin</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
              <widget class="PLSHelpIcon" name="label_2">
               <property name="toolTip">
                <string>chzzk.chat.limit.admin.tooltip</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>1</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_clip">
            <property name="text">
             <string>chzzk.clip.title</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="1">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>18</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="PLSRadioButton" name="radioButton_clip_allow" native="true">
               <property name="text" stdset="0">
                <string>chzzk.clip.allow</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="PLSRadioButton" name="radioButton_clip_not_allow" native="true">
               <property name="text" stdset="0">
                <string>chzzk.clip.not.allow</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="lineView" native="true"/>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QWidget" name="bottomButtonWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="useFor" stdset="0">
            <string notr="true">Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
           <property name="useFor" stdset="0">
            <string notr="true">Ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PLSLineEdit</class>
   <extends>QLineEdit</extends>
   <header>PLSEdit.h</header>
  </customwidget>
  <customwidget>
   <class>PLSLiveInfoDualWidget</class>
   <extends>QWidget</extends>
   <header>PLSLiveInfoBase.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PLSSelectImageButton</class>
   <extends>QLabel</extends>
   <header>PLSSelectImageButton.h</header>
  </customwidget>
  <customwidget>
   <class>PLSRadioButton</class>
   <extends>QWidget</extends>
   <header location="global">PLSRadioButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PLSHelpIcon</class>
   <extends>QLabel</extends>
   <header>PLSLabel.h</header>
  </customwidget>
  <customwidget>
   <class>PLSCheckBox</class>
   <extends>QWidget</extends>
   <header>PLSCheckBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PLSSearchCombobox</class>
   <extends>QLineEdit</extends>
   <header>PLSSearchCombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
